Gopkg.toml

```diff
@@ -435,7 +435,8 @@ required = [
 
 [[constraint]]
   name = "github.com/vishvananda/netlink"
-  revision = "b1e98597921720cee4147d6476110fc8fc56072d"
+  source = "github.com/DataDog/netlink.git"
+  branch = "lbernail/output-mark"
```

Comment:
next patch  pushes this to correct revision, so just put that change here vs changing and then changing it to correct value later.

Reply:
my preference would be to update the netlink library in its own commit.